generators/app/index.js

```diff
@@ -165,7 +168,7 @@ module.exports = class extends BaseGenerator {
     // update package.json in Ionic app
     const done = this.async();
     const packagePath = `${this.ionicAppName}/package.json`;
-    const packageJSON = this.fs.readJSON(packagePath);
+    const packageJSON = this.fs.readJSON(packagePath) || { devDependencies: [], dependencies: [], scripts: {}};
```

Comment:
Is this change needed? Is it so things don't fail if `package.json` isn't found? IMO, that should never happen.

Reply:
When I created a new Ionic project with `ionic4j`, package.json wasn't there.
Maybe I missed some steps.